howto/unstructured_dataset.md

```diff
@@ -0,0 +1,18 @@
+# Finetuning on an unstructured dataset
```

Comment:
Looks great! We could reference this guide in the other finetune documents too. For example like this:

```
See also: [Finetuning on an unstructured dataset](../unstructured_dataset.md)
```